### PR TITLE
Add docs for fetching translations

### PR DESCRIPTION
**Pull request by AdamGrzybkowski**
### Description

This PR adds docs to describe how to fetch the latest translations. [Here's the PR](https://github.com/Automattic/Gravatar-SDK-Android/pull/274) with the translation setup for reference.

Is there anything else worth mentioning? 

### Testing Steps

-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,17 +200,34 @@ We're using [kdoc](https://kotlinlang.org/docs/kotlin-doc.html) to document the
 
 Here is the most recent release of the [Gravatar-SDK-Android API documentation generated by Dokka](https://automattic.github.io/Gravatar-SDK-Android/dokka/current/index.html).
 
+
+## Translations
+
+Gravatar Android SDK uses [GlotPress](https://translate.wordpress.com/projects/gravatar/gravatar-android-sdk/gravatar-quick-editor/) to translate strings in the project.
```
Comment (hamorillo) [inline README.md]: ⛏ I would link the root "folder" in GlotPress.
```suggestion
Gravatar Android SDK uses [GlotPress](https://translate.wordpress.com/projects/gravatar/gravatar-android-sdk/) to translate strings in the project.
```

**Review by hamorillo — APPROVED** [review]
Maybe we can mention that strings are pulled from `trunk` regularly. But looks good to me.

File: README.md (hunk 1 above)
Comment (AdamGrzybkowski) [reply README.md]: ohh right! Thanks!

**Comment by AdamGrzybkowski** [comment]
>Maybe we can mention that strings are pulled from trunk regularly. But looks good to me.
 
 Yeah, I thought about it - but then this setup is in a completely different place so we could easily end up with outdated docs 🤔

**Comment by hamorillo** [comment]
I mentioned that because if some people just arrive at the project, they may have doubts about how to add new strings. But yeah, not a big deal.

**Pull request merged**